bucket/bt.json

```
@@ -0,0 +1,14 @@
+{
+    "version": "2.1",
+    "description": "Opens required browser based on configuration",
+    "homepage": "https://www.aloneguid.uk/projects/bt/",
+    "url": "https://www.aloneguid.uk/projects/bt/bin/bt-2.1.zip",
+    "bin": "bt.exe",
+    "hash": "cdf9827246daffc999ce476728a3ef578c00399054b1b6618d3b0fa7dfc41f23",
+    "shortcuts": [
+        [
+            "bt.exe",
+            "Browser Tamer"
+        ]
+    ]
+  }
```

```suggestion
    "checkver": "bin/bt-([\\d.]+)\\.zip",
    "autoupdate": {
        "url": "https://www.aloneguid.uk/projects/bt/bin/bt-$version.zip"
    }
  }

```
You can use "view source code" to find a web element that enables *Scoop* to track new versions.
Usually it is:
```
"checkver": {
    "url": "xxxxx",
    "regex": "xxxxx"
}
```
Since we can match the element on "homepage" (https://www.aloneguid.uk/projects/bt/), `checkver` can be reduced to one line.
For details, see: https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate

---

done